gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -275,6 +278,18 @@ public GsonBuilder enableComplexMapKeySerialization() {
     return this;
   }
 
+  /**
+   * Configure Gson to deserialize duplicate map key. By default, Gson throw a {@code JsonSyntaxException} when there are more than
+   * one same key.
+   *
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   * @since 2.8
+   */
+  public GsonBuilder enableDuplicateMapKeyDeserialization() {
```

Comment:
Maybe it would be good to rename it from "...Key" to "...Key<ins>s</ins>" since it applies to key**s** in general.
```suggestion
  public GsonBuilder enableDuplicateMapKeysDeserialization() {
```
What do you think?